packages/wpcom-checkout/src/checkout-line-items.tsx

```diff
@@ -707,6 +731,29 @@ function IntroductoryOfferCallout( {
 	return <DiscountCallout>{ text }</DiscountCallout>;
 }
 
+function PartnerLogo( {
+	coupon,
+	products,
+}: {
+	coupon: string;
+	products?: ResponseCartProduct[];
+} ): JSX.Element | null {
+	const translate = useTranslate();
+
+	if ( hasPartnerCoupon( coupon, products ) ) {
+		return (
+			<LineItemMeta>
+				<div>{ translate( 'Included in your IONOS plan' ) }</div>
+				<div>
+					<img src={ '/calypso/images/jetpack/partners/partner-logo-ionos.png' } alt="IONOS Logo" />
```

Comment:
Should we perhaps add max height/width values to the logo to match the suggested design? p1HpG7-dvB-p2#comment-51777

Reply:
I have added max width/height values for the partner logo in this case, but we should reconsider the styling in the future for different partners.